packages/store-ui/src/atoms/List/List.tsx

```diff
@@ -0,0 +1,56 @@
+import type { HTMLAttributes, ElementType } from 'react'
+import React, { forwardRef } from 'react'
+
+export type ListVariant = 'default' | 'description' | 'ordered' | 'unordered'
```

Comment:
Can you please change to a more appropriate naming than `default`? Maybe `regular`, or `inline`. Any name you can think of that conveys how it really behaves.

Reply:
Now that the default behavior for this component is rendering an unordered list, I think we can just remove the `default` variant :)

Reply:
yasss!